indra/literature/elsevier_client.py

```diff
@@ -233,20 +233,29 @@ def get_article(doi, output_format='txt'):
     return xml_string
 
 
-def extract_text(xml_string, contains=None):
+def extract_text(xml_string):
     """Get text from the body of the given Elsevier xml."""
+    paragraphs = extract_paragraphs(xml_string)
+    if paragraphs:
+        return '/n'.join(paragraphs) + '/n'
```

Comment:
This is probably meant to be \n, right?

Reply:
Wow, that’s pretty boneheaded. Yeah, my goal was to produce identical results to the existing implementation,